### ISSUE TITLE
v5.2.0 regexreplacelist not working

### ISSUE THREAD

**Issue by ghost** [e2guardian/e2guardian#451]
Hi, I think we have a bug in the redierct regexreplacelist / urlredirectregexplist.

I've got a really simple rule set up for testing, attempting to match anything, and redirect it to to google.com. 

Extract from my group config file:
`regexpreplacelist = 'name=redirect,path=/usr/local/etc/e2guardian/lists/urlredirectregexplist'`

Contents of my urlredirectregexplist
`.*"->"http://google.com"` 

Both this rule and anything more advanced dont appear to be working at all. 

e2guardian -v output:
```
e2guardian 5.2.0

Built with:  '--enable-sslmitm=yes' '--enable-pcre=yes'
```

**Comment by fredbcode** [comment]
SSLMITM ? google is a SSL website

**Comment by ghost** [comment]
Yes, running with SSLMITM. If I change my rule to either ` .*"->"http://google.com"` or ` .*"->"https://google.com"` I get the same result... Not working.

**Comment by fredbcode** [comment]
.*"->"http://google.com"
 it must be quotation marks
"regexp_pattern"->"url_to_redirect_to"

**Comment by ghost** [comment]
Sorry Fred. I missed off the first `"` when I copied the rule in. Can confirm my urlredirectregexplist looks like this `" .*"->"https://google.com"`  and the problem still remains,

**Comment by fredbcode** [comment]
Can you try something less global "*bing.com"->"https://google.com"

**Comment by fredbcode** [comment]
".*bing.com" better

**Comment by ghost** [comment]
No change, still no redirect and bing.com showing in the logs as normal

**Comment by fredbcode** [comment]
Hum, do you have if(fullurlin,redirect) return setredirect in storyboard ?

**Comment by ghost** [comment]
🙌 yes, that's fixed things as per bug #417 - apologies for not digging diligently enough to find that in the first place, and thanks very much for your help Fred.
